bin/build/populate-plugin-by-product.sh

```diff
@@ -4,13 +4,13 @@ trap 'set +ex' EXIT
 
 case "$1" in
   jira)
-    echo "::set-env name=PLUGIN::jira-slack-server-integration/jira-slack-server-integration-plugin"
+    echo "PLUGIN=jira-slack-server-integration/jira-slack-server-integration-plugin" >> $GITHUB_ENV
     ;;
   confluence)
-    echo "::set-env name=PLUGIN::confluence-slack-server-integration-plugin"
+    echo "PLUGIN=confluence-slack-server-integration-plugin" >> $GITHUB_ENV
     ;;
   bitbucket)
-    echo "::set-env name=PLUGIN::bitbucket-slack-server-integration-plugin"
+    echo "PLUGIN=bitbucket-slack-server-integration-plugin" >> $GITHUB_ENV
```

Comment:
Github changed the way of setting environment variables withing a workflow run to a more secure approach: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#environment-files